package.json

```diff
@@ -14,6 +14,7 @@
     "build": "nx run core:build",
     "build:server": "http-server dist",
     "dev": "nx run core:dev",
+    "dev:local": "WEBPACK_DONE='rsync -rvc --progress --delete --compress-level=9 build store.bcdev:/opt/bigcommerce_app/vagrant_code/vendor/bower_components/checkout/' npm run dev",
```

Comment:
Added this as a convenience method to run in local environment.

Reply:
Perhaps we should avoid calling it `dev:local` because `local` typically indicates a local machine for most developers.
Maybe we could use `dev:vm` or something more relevant to our specific context.

Reply:
I like `dev:vm`, will update it. Thanks.